src/script/lua_api/l_env.cpp

```diff
@@ -436,20 +437,30 @@ int ModApiEnv::l_place_node(lua_State *L)
 		lua_pushboolean(L, false);
 		return 1;
 	}
+
 	// Create item to place
 	std::optional<ItemStack> item = ItemStack(ndef->get(n).name, 1, 0, idef);
 	// Make pointed position
 	PointedThing pointed;
 	pointed.type = POINTEDTHING_NODE;
 	pointed.node_abovesurface = pos;
 	pointed.node_undersurface = pos + v3s16(0,-1,0);
+
+	ServerActiveObject *placer = nullptr;
+
+	if (! lua_isnoneornil(L, 3)) {
+		ObjectRef *ref = checkObject<ObjectRef>(L, 3);
+		placer = ObjectRef::getobject(ref);
+	}
+
 	// Place it with a NULL placer (appears in Lua as nil)
```

Comment:
```suggestion
	// Place it with a nullptr placer (appears in Lua as nil)
```

(also elsewhere)

Reply:
Resolved in 391f46c62105d536b0fef43a62767c10b95a21a1

Reply:
Nah, in 52b07aee1 instead